custom/icds_reports/sqldata/exports/growth_tracker_report.py

```diff
@@ -0,0 +1,166 @@
+from dateutil.relativedelta import relativedelta
+
+from corehq.apps.locations.models import SQLLocation
+from custom.icds_reports.models import ChildHealthMonthlyView
+from custom.icds_reports.sqldata.base import IcdsSqlData
+from custom.icds_reports.utils import get_status, calculate_date_for_age, \
+    current_month_stunting_column, \
+    current_month_wasting_column, phone_number_function, india_now
+from custom.icds_reports.utils.mixins import ExportableMixin
+
+
+class GrowthTrackerExport(ExportableMixin, IcdsSqlData):
+    title = 'Child Growth Tracking'
+    table_name = 'child_health_monthly_view'
+
+    def get_excel_data(self, location):
+
+        def _check_case_presence(case_id, column, data_dict):
+            return data_dict[case_id][column] if case_id in data_dict.keys() else "N/A"
+
+        def _fetch_data(filters, order_by, case_by_grouping=False):
+            query_set = ChildHealthMonthlyView.objects.filter(**filters).order_by(*order_by)
+            data_month = query_set.values('state_name', 'district_name', 'block_name', 'supervisor_name',
+                                          'awc_name', 'awc_site_code', 'person_name', 'dob', 'mother_name',
+                                          'mother_phone_number', 'pse_days_attended', 'lunch_count',
+                                          'current_month_nutrition_status',
+                                          current_month_wasting_column(self.beta),
+                                          current_month_stunting_column(self.beta), 'case_id')
+            if case_by_grouping is True:
+                data_month = {item['case_id']: item for item in data_month}
+            return data_month
+
+        filters = {
+            "month": self.config['month'],
+            "age_in_months__lte": 72,
```

Comment:
Can you confirm what do we want to fill in cell for the indicator which do not apply for  the children. Like stunting does not apply for 5-6 years of children.

Reply:
Talked this regarding the GS team, they said let's keep it as same as dashboard

Reply:
Will this also take care of the case where a past month's data is not available?

Reply:
It will automatically fill N/A for all of them